### PR TITLE
[#270] Implement no_debug_call rule.

### PR DESCRIPTION
**Pull request by bullno1**
Should I add the rule to elvis.config too?

### CODE REVIEW

**Comment by elbrujohalcon** [comment]
Add it to elvis.config please (with just io:format and ct:pal) and also to the rules page in the project wiki, if you haven't done that yet

File: src/elvis_style.erl [hunk 1]
```diff
@@ -502,6 +506,33 @@ max_function_length(Config, Target, RuleConfig) ->
...
+
+-spec no_debug_call(elvis_config:config(),
+                    elvis_file:file(),
+                    no_debug_call_config()) ->
+    [elvis_result:item()].
+no_debug_call(Config, Target, RuleConfig) ->
+    IgnoreModules = maps:get(ignore, RuleConfig, []),
+    {Root, _} = elvis_file:parse_tree(Config, Target),
+    ModuleName = elvis_code:module_name(Root),
+    DefaultDebugFuns = [{ct, pal},
```
Comment (elbrujohalcon) [inline src/elvis_style.erl]: shouldn't it be `{ct, pal, 1}, {ct, pal, 2}`, like `{io, format, …}` ?

**Comment by elbrujohalcon** [comment]
:+1: from me, although let's wait for @amilkr to review and merge it.

**Comment by jfacorro** [comment]
@bullno1 Hi, thank you for your awesome contribution. The `config/elvis.config` configuration file is the one where all default `Elvis` rules are kept. There is also the wiki page that lists all [Default Rules in Elvis Webhook](https://github.com/inaka/elvis/wiki/Default-Rules-in-Elvis-Webhook). Could you please add the rule in that list as well?

- [ ] Add `no_debug_call` rule to `config/elvis.config`.
- [ ] Update wiki page [Default Rules in Elvis Webhook](https://github.com/inaka/elvis/wiki/Default-Rules-in-Elvis-Webhook).

**Comment by elbrujohalcon** [comment]
@bullno1 any updates on this?

**Comment by bullno1** [comment]
Sorry for taking so long, it's done now.

File: temp_file_test [hunk 2]
```diff
@@ -0,0 +1 @@
+sdsds
```
Comment (elbrujohalcon) [inline temp_file_test]: please remove this file, thanks.

**Pull request merged**